plopfile.js

```diff
@@ -165,7 +165,7 @@ module.exports = (plop) => {
           template: [
             "rootProject.name='example'",
             "",
-            'apply from: file("${rootDir}/node_modules/react-native-test-app/test-app.gradle")',
+            `apply from: file("${exclusive ? "" : "../"}node_modules/react-native-test-app/test-app.gradle")`,
```

Comment:
I think it would be better to use `findNodeModules` here instead. Then you don't have to differentiate between exclusive and non-exclusive modes.

Reply:
Isn't this file something the consumer would write? Do they have access to that function here?

Reply:
That's a good point. There is an accessible utility function in the `root/android/test-app-util.gradle` file. Another question if we want it to be a part of the public API of the library.. Let's keep it as it is for now.